cirq-core/cirq/vis/circuit_to_latex_quantikz.py

```diff
@@ -0,0 +1,710 @@
+
+
+# =============================================================================
+# Default Preamble Template (physics.sty removed)
+# =============================================================================
+DEFAULT_PREAMBLE_TEMPLATE = r"""
+\documentclass[preview, border=2pt]{standalone}
+% Core drawing packages
+\usepackage{tikz}
+\usetikzlibrary{quantikz} % Loads the quantikz library (latest installed version)
+% Optional useful TikZ libraries
+\usetikzlibrary{fit, arrows.meta, decorations.pathreplacing, calligraphy}
+% Font encoding and common math packages
+\usepackage[T1]{fontenc}
+\usepackage{amsmath}
+\usepackage{amsfonts}
+\usepackage{amssymb}
+% \usepackage{physics} % Removed
+"""
+
+# =============================================================================
+# Default Style Definitions
+# =============================================================================
+_Pauli_gate_style = r"style={fill=blue!20}"
+_green_gate_style = r"style={fill=green!20}"
+_yellow_gate_style = r"style={fill=yellow!20}"  # For H
+_orange_gate_style = r"style={fill=orange!20}"  # For FSim, ISwap, etc.
+_gray_gate_style = r"style={fill=gray!20}"  # For Measure
+_noisy_channel_style = r"style={fill=red!20}"
+
+GATE_STYLES_COLORFUL1 = {
+    "H": _yellow_gate_style,
+    "_PauliX": _Pauli_gate_style,  # cirq.X(q_param)
+    "_PauliY": _Pauli_gate_style,  # cirq.Y(q_param)
+    "_PauliZ": _Pauli_gate_style,  # cirq.Z(q_param)
+    "X": _Pauli_gate_style,  # For XPowGate(exponent=1)
+    "Y": _Pauli_gate_style,  # For YPowGate(exponent=1)
+    "Z": _Pauli_gate_style,  # For ZPowGate(exponent=1)
+    "X_pow": _green_gate_style,  # For XPowGate(exponent!=1)
+    "Y_pow": _green_gate_style,  # For YPowGate(exponent!=1)
+    "Z_pow": _green_gate_style,  # For ZPowGate(exponent!=1)
+    "H_pow": _green_gate_style,  # For HPowGate(exponent!=1)
+    "Rx": _green_gate_style,
+    "Ry": _green_gate_style,
+    "Rz": _green_gate_style,
+    "PhasedXZ": _green_gate_style,
+    "FSimGate": _orange_gate_style,
+    "FSim": _orange_gate_style,  # Alias for FSimGate
+    "ISwap": _orange_gate_style,  # For ISwapPowGate(exponent=1)
+    "iSWAP_pow": _orange_gate_style,  # For ISwapPowGate(exponent!=1)
+    "CZ_pow": _orange_gate_style,  # For CZPowGate(exponent!=1)
+    "CX_pow": _orange_gate_style,  # For CNotPowGate(exponent!=1)
+    "CXideal": "",  # No fill for \ctrl \targ, let quantikz draw default
+    "CZideal": "",  # No fill for \ctrl \control
+    "Swapideal": "",  # No fill for \swap \targX
+    "Measure": _gray_gate_style,
+    "DepolarizingChannel": _noisy_channel_style,
+    "BitFlipChannel": _noisy_channel_style,
+    "ThermalChannel": _noisy_channel_style,
+}
+
+
+# =============================================================================
+# Cirq to Quantikz Conversion Class
+# =============================================================================
+class CirqToQuantikz:
```

Comment:
nit: I think I would prefer CircuitToQuantikz, but maybe that's just a personal preference.